app/scripts/metamask-controller.js

```diff
@@ -1216,7 +1202,7 @@ module.exports = class MetamaskController extends EventEmitter {
    * @property {string} depositType - An abbreviation of the type of crypto currency to be deposited.
    */
   createShapeShiftTx (depositAddress, depositType) {
```

Comment:
Yeah again here is a great example of where we could just have `createShapeShift: this.shapeshiftController.createTransaction.bind(this.shapeshiftController),` in the `getAPI()` method, instead of this entire function.